frontend/packages/console-app/src/components/detect-perspective/useValuesForPerspectiveContext.ts

```diff
@@ -13,12 +14,19 @@ export const useValuesForPerspectiveContext = (): [
   boolean,
 ] => {
   const perspectiveExtensions = useExtensions<Perspective>(isPerspective);
-  const [perspective, setPerspective, loaded] = useUserSettingsCompatibility<PerspectiveType>(
-    LAST_PERSPECTIVE_USER_SETTINGS_KEY,
-    LAST_PERSPECTIVE_LOCAL_STORAGE_KEY,
-    '',
-  );
+  const [lastPerspective, setLastPerspective, lastPerspectiveLoaded] = useUserSettingsCompatibility<
+    PerspectiveType
+  >(LAST_PERSPECTIVE_USER_SETTINGS_KEY, LAST_PERSPECTIVE_LOCAL_STORAGE_KEY, '');
+  const [preferredPerspective, preferredPerspectiveLoaded] = usePreferredPerspective();
+
+  const loaded = lastPerspectiveLoaded && preferredPerspectiveLoaded;
+
+  const perspective =
+    loaded && preferredPerspective && preferredPerspective !== 'latest'
```

Comment:
Const see below (should be replace all 'latest')

```suggestion
    loaded && preferredPerspective && preferredPerspective !== PREFERRED_PERSPECTIVE_USER_SETTING_LATEST
```